<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
    <item row="3" column="0">
     <widget class="QGroupBox" name="gridGroupBox">
      <property name="title">
       <string>复位控制</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>复位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonReset">
           <property name="text">
            <string>全部关闭</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QGroupBox" name="GroupBoxValve">
      <property name="title">
       <string>阀门控制</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="2" column="1" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>水泵</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonWaterPump">
           <property name="text">
            <string>打开</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonWaterPumpOff">
           <property name="text">
            <string>关闭</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonWaterLED">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>38</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>38</width>
             <height>38</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
border-image: url(:/img/指示灯-红.png);
}
QPushButton::checked
{
border-image: url(:/img/指示灯-绿.png);
}
QPushButton::unchecked
{
border-image: url(:/img/指示灯-红.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>粉阀</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonPowderPump">
           <property name="text">
            <string>打开</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonPowderPumpOff">
           <property name="text">
            <string>关闭</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonPowderLED">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>38</width>
             <height>38</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>38</width>
             <height>38</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
border-image: url(:/img/指示灯-红.png);
}
QPushButton::checked
{
border-image: url(:/img/指示灯-绿.png);
}
QPushButton::unchecked
{
border-image: url(:/img/指示灯-红.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="GroupBoxSeries">
      <property name="title">
       <string>串口设置</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>选择串口</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxSerial"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSerial">
           <property name="minimumSize">
            <size>
             <width>87</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>87</width>
             <height>28</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
border-image: url(:/img/btncheckoff2.png);
}
QPushButton::checked
{
border-image: url(:/img/btncheckon2.png);
}
QPushButton::unchecked
{
border-image: url(:/img/btncheckoff2.png);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>87</width>
             <height>28</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="windowTopButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>38</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>38</width>
           <height>38</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
border-image: url(:/img/窗口置顶.png);
}
QPushButton::checked
{
border-image: url(:/img/已置顶.png);
}
QPushButton::unchecked
{
border-image: url(:/img/窗口置顶.png);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="4" colspan="2">
     <widget class="QGroupBox" name="gridGroupBox">
      <property name="title">
       <string>发动机控制</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="0">
        <widget class="QGroupBox" name="gridGroupBox">
         <property name="title">
          <string>油门控制</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="3" column="3">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QSlider" name="verticalSliderEngine">
              <property name="minimumSize">
               <size>
                <width>68</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">/* 绿色宽版垂直滑块样式 */
QSlider::groove:vertical {
    background: #37474F;
    width:800px;  /* 加宽轨道 */
}

QSlider::handle:vertical {
    background: rgb(255, 255, 255);
    width: 50px;  /* 加大手柄 */
    height: 20px;

}

QSlider::sub-page:vertical {
    background: #455A64;
    border: 1px solid #1B5E20;
    border-radius: 15px;
}

QSlider::add-page:vertical {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                           stop:0 #2E7D32, stop:0.5 #4CAF50, stop:1 #81C784);
    border: 1px solid #37474F;
    border-radius: 15px;
}

QSlider::handle:vertical:hover {
    background: qradialgradient(cx:0.5, cy:0.5, radius:0.5, 
                           fx:0.5, fy:0.5, 
                           stop:0.8 white, stop:1.0 #81C784);
    border-color: #388E3C;
}

QSlider::handle:vertical:pressed {
    background: qradialgradient(cx:0.5, cy:0.5, radius:0.5, 
                           fx:0.5, fy:0.5, 
                           stop:0.8 white, stop:1.0 #1B5E20);
    border-color: #1B5E20;
}</string>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxEngineValue">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0" rowspan="3" colspan="3">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="pushButtonEngineControl">
              <property name="styleSheet">
               <string notr="true">QPushButton::checked
{
	background-color: rgb(255, 85, 0);
}</string>
              </property>
              <property name="text">
               <string>开启控制</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonEngineStart">
              <property name="styleSheet">
               <string notr="true">QPushButton::checked
{
	background-color: rgb(85, 255, 127);
}</string>
              </property>
              <property name="text">
               <string>启动</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonEngineStop">
              <property name="styleSheet">
               <string notr="true">QPushButton::checked
{
	background-color: rgb(85, 255, 127);
}</string>
              </property>
              <property name="text">
               <string>停车/散热</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonEngineEmergentStop">
              <property name="styleSheet">
               <string notr="true">QPushButton::checked
{
	background-color: rgb(85, 255, 127);
}</string>
              </property>
              <property name="text">
               <string>紧急停车</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="0" colspan="4">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>转速：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QGroupBox" name="gridGroupBox_2">
         <property name="title">
          <string>测试油泵</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonTestOilPump">
            <property name="text">
             <string>测试油泵(短时)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonOilPumpLED">
            <property name="minimumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
border-image: url(:/img/指示灯-红.png);
}
QPushButton::checked
{
border-image: url(:/img/指示灯-绿.png);
}
QPushButton::unchecked
{
border-image: url(:/img/指示灯-红.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>电压：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
